push_notifications/apns_async.py

```diff
@@ -397,10 +397,9 @@ async def _send_bulk_request(
 
 	aps_kwargs = {}
 	if mutable_content:
-		aps_kwargs["mutable-content"] = mutable_content
+		aps_kwargs["mutable-content"] = 1
```

Comment:
right this is to help align with the implementation of apns.py?

Reply:
Update: this is looks great! Just confirmed with the tests, no additional change required. Thanks for the MR 👍🏼